styles/styles.css

```diff
@@ -0,0 +1,776 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+/*////////////////////////
+
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --text-color-accent: #5722CD;
+    --header-bg: #4845E4;
+    --button-pink: #ED4385;
+    --main-text: #0C0C0C;
+    --header-text: #F2F0FF;
+    --background: #F3F6F9;
+    --service-tile-text: #7A7A7A;
+    --basic-white: #FFFFFF;
+}
+
+body {
+    /*It seems that I have to add this when using the noise overlay, otherwise nothing is clickable and the body is laying on top*/
+    z-index: -3;
+    position: relative;
+
+}
+
+/*Background noise image*/
+.outer-header::after,
+.outer-contact-form::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    opacity: 0.5;
+    background: url(../assets/noise-texture.png) repeat;
+    background-size: auto;
+    z-index: -1;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+body {
+    font-family: "Roboto", sans-serif;
+    font-weight: 500;
+}
+
+/*--Heading typography--*/
+h1,
+h2,
+h3,
+h4,
+h5 {
+    font-family: "Merriweather", serif;
+    font-weight: 700;
+    text-align: center;
+}
+
+h1,
+h5 {
+    font-size: 1.7rem;
+    display: block;
+    width: 100%;
+    color: var(--header-text);
+}
+
+h2 {
+    color: var(--text-color-accent);
+    font-size: 1.2rem;
+    margin: 0 0 0.7em 0;
+}
+
+h3 {
+    text-align: left;
+    font-size: 1.5rem;
+    margin: 0 0 1em 0;
+}
+
+h4 {
+    color: var(--text-color-accent);
+    font-size: 1.1rem;
+    text-align: left;
+    margin: 0.5em 0;
+}
+
+/*--Paragraph typography--*/
+p {
+    padding: 1em 0 1.8em 0;
+}
+
+.header-content p {
+    color: var(--header-text);
+    font-size: 1.3rem;
+    font-weight: 300;
+}
+
+.introduction-text p {
+    text-align: center;
+    padding: 4em 0 0 0;
+    position: relative;
+}
+
+.info-block p {
+    font-size: 0.9rem;
+    color: var(--service-tile-text);
+    padding: 0.5em 0;
+    font-weight: 400;
+}
+
+.inner-contact p {
+    color: var(--header-text);
+    font-weight: 300;
+}
+
+footer p {
+    font-size: 0.7rem;
+    font-weight: 300;
+    color: var(--basic-white);
+    padding: 5px 0;
```

Comment:
gebruik geen px en em door elkaar in een stylesheet. Kies voor 1 van de 2 en blijf bij die keuze.